src/ontology/cl-edit.owl

```diff
@@ -16604,25 +16604,28 @@ SubClassOf(obo:CL_0002302 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0006909))
 
 # Class: obo:CL_0002303 (pigmented ciliary epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15106942"^^xsd:string) obo:IAO_0000115 obo:CL_0002303 "A ciliated epithelial cell of the retina, this cell type uptakes sodium chloride and passes it to non-pigmented ciliary epithelial cells."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15106942"^^xsd:string) obo:IAO_0000115 obo:CL_0002303 "A cell that is part of pigmented ciliary epithelium. This cell type uptakes sodium and chloride ions from stromal interstitium and passes the ions to non-pigmented ciliary epithelial cells via gap junctions."^^xsd:string)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0002303 <https://orcid.org/0000-0003-1980-3228>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002303 "2010-09-13T04:09:27Z"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002303 "FMA:70606"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002303 "PE"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://doi.org/10.1016/j.exer.2003.09.021"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_0002303 "PE cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://doi.org/10.1016/j.exer.2003.09.021"^^xsd:string) rdfs:comment obo:CL_0002303 "In a consensus model of unidirectional secretion, aqueous humour is formed by transferring solute, principally NaCl, from stroma to posterior chamber, with water passively following the osmotic gradient. Unidirectional secretion comprises three sequential steps. First, NaCl is transferred from the stromal interstitium into the pigmented ciliary epithelial (PE) cells… Second, solute passes from the PE into the non-pigmented ciliary epithelial (NPE) cells through gap junctions. Finally, NPE cells release Na+ and Cl− into the aqueous humour…"^^xsd:string)
```

Comment:
Looking at the usage of ellipsis, when the ommited text contains a period, four dots should be used.